extmod/nimble/modbluetooth_nimble.c

```diff
@@ -289,8 +289,15 @@ STATIC int gap_event_cb(struct ble_gap_event *event, void *arg) {
                 mp_bluetooth_gatts_on_indicate_complete(event->notify_tx.conn_handle, event->notify_tx.attr_handle, event->notify_tx.status == BLE_HS_EDONE ? 0 : event->notify_tx.status);
             }
         }
-    }
 
+        case BLE_GAP_EVENT_MTU: {
+            if (event->mtu.channel_id == BLE_L2CAP_CID_ATT) {
```

Comment:
Did you find a reference for using `BLE_L2CAP_CID_ATT` here? Should there be a log if it's fired with a different `channel_id` or is this normal to be ignored behavior?

Reply:
I see it fired with different CIDs at other times.

Reply:
That's really curious, I wonder if that's causing intermittent issues in my current implementation which doesn't have this check (nice work finding this). I see there's a similar check in one of the nimble test apps.